cfme/fixtures/post_appliance_upgrade.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup("cfme")
+    group.addoption(
+        "--post-upgrade",
+        dest="post_upgrade",
+        action="store_true",
+        default=False,
+        help="Enable upgrade an appliance before the first test is run.",
+    )
+    group.addoption(
+        "--upgrade-to",
+        dest="upgrade_to",
+        action="store_true",
```

Comment:
I don't think this is the behavior you want here, you want the user to be able to pass a string.